src/tests/test_sexual_behaviour.py

```diff
@@ -16,17 +17,26 @@ def check_prob_sums(sex, trans_matrix):
     (dim, _) = trans_matrix.shape
     for i in range(0, dim):
         ages = np.array([15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65])
+        sexes = np.array([sex]*len(ages))
+        pop = pd.DataFrame({"age": ages, "sex": sexes})
+        SBM.init_risk_factors(pop)
+        assert len(pop["rred"]) == 11
+        print(pop["new_partner_factor"])
+        for r in pop["new_partner_factor"]:
+            assert 0 < r <= 2
```

Comment:
Minor, personal preference:
```suggestion
        assert all((0 < pop["new_partner_factor"]) & (pop["new_partner_factor"] <= 2))
```